.circleci/config.yml

```diff
@@ -564,6 +586,10 @@ workflows:
                                 - main
                                 - /release\/.*/
                                 - next
+            -   snyk-test:
+                    requires:
+                        - dashboards-build
```

Comment:
It would be good to re-add the comment from the closed PR about waiting until the last package is built before running the snyk test.